jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -818,12 +818,10 @@ ecma_uint32_to_utf8_string (uint32_t value, /**< value to convert */
  *
  * @return number - result of conversion.
  */
-ecma_number_t
+ecma_number_t __attr_always_inline___
```

Comment:
It doesn't make any difference. The binary sizes are the same.
There are the results without this attributes:
(It's possible, that this gain is only noise)

                      Benchmark |                   Perf(+ is better)
                     3d-cube.js |   2.645s ->   2.617s :  +1.071%  (+-0.824%) : [+]
         access-binary-trees.js |   1.410s ->   1.430s :  -1.418%  (+-0.000%) : [-]
             access-fannkuch.js |   7.985s ->   7.997s :  -0.146%  (+-0.460%) : [≈]
                access-nbody.js |   3.083s ->   3.115s :  -1.027%  (+-0.526%) : [-]
    bitops-3bit-bits-in-byte.js |   1.737s ->   1.713s :  +1.344%  (+-0.688%) : [+]
         bitops-bits-in-byte.js |   2.660s ->   2.647s :  +0.501%  (+-0.929%) : [≈]
          bitops-bitwise-and.js |   3.682s ->   3.780s :  -2.671%  (+-1.079%) : [-]
       controlflow-recursive.js |   0.890s ->   0.887s :  +0.374%  (+-0.955%) : [≈]
                  crypto-aes.js |   4.727s ->   4.695s :  +0.670%  (+-0.512%) : [+]
                  crypto-md5.js |  21.542s ->  21.543s :  -0.007%  (+-0.451%) : [≈]
                 crypto-sha1.js |   9.767s ->   9.760s :  +0.068%  (+-0.448%) : [≈]
           date-format-tofte.js |   2.233s ->   2.217s :  +0.746%  (+-0.846%) : [≈]
           date-format-xparb.js |   1.085s ->   1.077s :  +0.768%  (+-1.137%) : [≈]
                 math-cordic.js |   3.005s ->   2.980s :  +0.832%  (+-0.829%) : [+]
           math-partial-sums.js |   1.665s ->   1.680s :  -0.901%  (+-0.546%) : [-]
          math-spectral-norm.js |   1.513s ->   1.487s :  +1.762%  (+-0.787%) : [+]
                string-fasta.js |   4.542s ->   4.557s :  -0.330%  (+-0.611%) : [≈]
                Geometric mean: |                 Speed up: 0.102% (+-0.179%) : [≈]